### DISCUSSION TITLE
How does one offset the time of a single time-series (Cloudwatch) so that the graphs align properly?

### DISCUSSION THREAD

**Discussion by contentfree** [grafana/grafana#64838] [Dashboards, Panels & Visualizations]
I have a dashboard that shows transactions per second and DB CPU. The TPS is from a Prometheus query. The DB CPU comes from Cloudwatch metrics and the time is always two minutes earlier than my transactions per second information. How do I shift one of these time so that the graph is lined up properly?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
